triggers/incident-triggered/handler.py

```diff
@@ -0,0 +1,27 @@
+from relay_sdk import Interface, WebhookServer
+from quart import Quart, request, render_template_string
+
+import logging, sys
+
+relay = Interface()
+app = Quart('incident-triggered')
+
+logging.getLogger().setLevel(logging.INFO)
+
+@app.route('/', methods=['POST'])
+async def handler():
+    data = await request.form
+
+    event = { 'incident': 0 }
+
+    # handle incident_id a bit special since it's the main key, the rest of the
```

Comment:
I think this comment is no longer accurate.

Reply:
good catch thanks